apps-rendering/src/components/headlineByline.tsx

```diff
@@ -0,0 +1,85 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import { background } from '@guardian/common-rendering/src/editorialPalette/background';
+import type { ArticleFormat } from '@guardian/libs';
+import { ArticleSpecial } from '@guardian/libs';
+import { headline, remSpace, textSans } from '@guardian/source-foundations';
+import type { Option } from '@guardian/types';
+import { map, withDefault } from '@guardian/types';
+import { pipe } from 'lib';
+import type { ReactElement, ReactNode } from 'react';
+import { getHref } from 'renderer';
+
+const toReact = (format: ArticleFormat) => {
+	return function getReactNode(node: Node, index: number): ReactNode {
+		switch (node.nodeName) {
+			case 'A':
+				return (
+					<a
+						href={withDefault('')(getHref(node))}
+						key={`anchor-${index}`}
+					>
+						{node.textContent ?? ''}
+					</a>
+				);
+			case 'SPAN':
+				return Array.from(node.childNodes).map(toReact(format));
+			case '#text':
+				return node.textContent;
+		}
+	};
+};
+
+const renderText = (
+	format: ArticleFormat,
+	byline: DocumentFragment,
+): ReactNode =>
+	Array.from(byline.childNodes).map((node, i) => toReact(format)(node, i));
+
+const headlineBox = (format: ArticleFormat): SerializedStyles => css`
+	${format.theme === ArticleSpecial.Labs
+		? textSans.large({ lineHeight: 'regular' })
+		: headline.xxsmall({
+				fontWeight: 'regular',
+				lineHeight: 'loose',
+		  })}
+	font-style: italic;
+
+	a {
+		color: inherit;
+		text-decoration: none;
+		:hover {
```

Comment:
It does but I'm not sure if `hover` is semantically accurate since technically users will tap on the link instead of hovering over it with a cursor...

Reply:
> instead of hovering over it with a cursor...
Is this an assumption that devices running the app don't have cursors? Some of them do 🙂: https://developer.apple.com/design/human-interface-guidelines/ios/user-interaction/pointers/